source/read-document-playground.txt

```diff
@@ -61,6 +61,10 @@ in the MongoDB Manual.
 
 .. include:: /includes/run-playground.rst
 
+You may edit the returned |json| document.
```

Comment:
Can this be, at least:
While you are viewing one document, you can also edit it. 
Then add a stem sentence to the include procedure. As written, this is a procedure without its own heading or stem sentence, tacked onto another topic, and it is confusing. Maybe, the title can be changed to : Read one and edit? Something's off here and if this section is what you added, and it is in scope, consider one of these:
- changing the title to indicate that edits are also possible
- saying: while you view, you can also edit
- adding a stem sentence before the included steps for how to edit.

Reply:
This was updated to: `You may edit any |json| document returned from a |findOne| or |findAny| operation.`.